--- jdresolve-0.6.1/Makefile.orig	Mon Oct 16 11:43:31 2000
+++ jdresolve-0.6.1/Makefile.in	Sun Jan  6 10:18:15 2002
@@ -12,12 +12,12 @@
 	for FILE in AUTHORS README BUGS CHANGELOG CREDITS INSTALL TODO; do lynx --dump "$$FILE.html" | perl -ne "! /^Looking/ and print" > "$$FILE"; done
 
 install:
-	$(INSTALL) -m 755 -o bin -g bin jdresolve $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 755 -o bin -g bin rhost $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 jdresolve $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 rhost $(DESTDIR)$(BINDIR)
 
 	mkdir -p $(DESTDIR)$(MANDIR)
-	$(INSTALL) -m 644 -o bin -g bin jdresolve.1 $(DESTDIR)$(MANDIR)
-	$(INSTALL) -m 644 -o bin -g bin rhost.1 $(DESTDIR)$(MANDIR)
+	$(INSTALL) -m 644 jdresolve.1 $(DESTDIR)$(MANDIR)
+	$(INSTALL) -m 644 rhost.1 $(DESTDIR)$(MANDIR)
 
 uninstall:
 	rm $(DESTDIR)$(BINDIR)/jdresolve
