--- jdresolve-0.5.2/Makefile.wiget	Sun Aug 22 15:10:24 1999
+++ jdresolve-0.5.2/Makefile	Fri Apr 14 13:03:30 2000
@@ -1,12 +1,14 @@
 prefix=/usr
 INSTALL=/usr/bin/install
 BINDIR=$(prefix)/bin
+USER=bin
+GROUP=bin
 
 none: 
 
 install:
-	$(INSTALL) -m 555 -o bin -g bin jdresolve $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 555 -o bin -g bin jdresolve-dumpdb $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 555 -o bin -g bin jdresolve-mergedb $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 555 -o bin -g bin jdresolve-unresolved $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 555 -o bin -g bin rhost $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 555 -o $(USER) -g $(GROUP) jdresolve $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 555 -o $(USER) -g $(GROUP) jdresolve-dumpdb $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 555 -o $(USER) -g $(GROUP) jdresolve-mergedb $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 555 -o $(USER) -g $(GROUP) jdresolve-unresolved $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 555 -o $(USER) -g $(GROUP) rhost $(DESTDIR)$(BINDIR)
